File: app/src/main/java/com/bubelov/coins/util/OnSeekBarChangeAdapter.java
package com.bubelov.coins.util;

import android.widget.SeekBar;

/**
 * Author: Igor Bubelov
 */

public abstract class OnSeekBarChangeAdapter implements SeekBar.OnSeekBarChangeListener {
    @Override
    public void onProgressChanged(SeekBar seekBar, int progress, boolean fromUser) {
        // Nothing to do here
    }

    @Override
    public void onStartTrackingTouch(SeekBar seekBar) {
        // Nothing to do here
    }

    @Override
    public void onStopTrackingTouch(SeekBar seekBar) {
        // Nothing to do here
    }
}

File: app/src/main/java/com/bubelov/coins/util/OnSeekBarChangeAdapter.kt
package com.bubelov.coins.util

import android.widget.SeekBar

/**
 * Author: Igor Bubelov
 */

abstract class OnSeekBarChangeAdapter : SeekBar.OnSeekBarChangeListener {
    override fun onProgressChanged(seekBar: SeekBar, progress: Int, fromUser: Boolean) {
        // Nothing to do here
    }

    override fun onStartTrackingTouch(seekBar: SeekBar) {
        // Nothing to do here
    }

    override fun onStopTrackingTouch(seekBar: SeekBar) {
        // Nothing to do here
    }
}